operating/manage-compliance-operator/compliance-operator-rhacs.adoc

```diff
@@ -7,7 +7,16 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-You can configure {product-title-short} to use the Compliance Operator for compliance reporting and remediation with {ocp} clusters. Results from the Compliance Operator can be reported in the {product-title-short} Compliance Dashboard.
+You can configure {product-title-short} to use the Compliance Operator for compliance reporting and remediation with {ocp} clusters. Results from the Compliance Operator are reported in the {product-title-short} Compliance Dashboard.
+
+The Compliance Operator automates the review of numerous technical implementations and compares them with certain aspects of industry standards, benchmarks and baselines. 
```

Comment:
```suggestion
The Compliance Operator automates the review of numerous technical implementations and compares them with certain aspects of industry standards, benchmarks, and baselines. 
```

Reply:
https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series